src/View/Cell.php

```diff
@@ -16,6 +16,7 @@
 
 use BadMethodCallException;
 use Cake\Cache\Cache;
+use Cake\Core\Configure;
```

Comment:
I'll clean that up. I was considering using App.namespace but came up with a better way.